cmd/cli/completion.go

```diff
@@ -76,6 +76,31 @@ func resourceNamesCompletionFunc(gvk schema.GroupVersionKind) func(cmd *cobra.Co
 	}
 }
 
+// resourceKindCompletionFunc returns a function that can be used as a completion function for Flux resource kinds.
+func resourceKindCompletionFunc() func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+	return func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+		kinds := []string{
```

Comment:
Is this list only supposed to have suspendable objects? If yes, I'd add this to the function comment. If not, I'd add the remaining Flux kinds to the list.